packages/pass-style/test/safe-promise.test.js

```diff
@@ -50,10 +49,7 @@ test('safe promise loophole', t => {
       value: 'Promise p4',
       enumerable: true,
     });
-    t.throws(() => passStyleOf(harden(p4)), {
-      message:
-        'Own @@toStringTag must not be enumerable: {"configurable":false,"enumerable":true,"value":"Promise p4","writable":false}',
-    });
+    t.is(passStyleOf(harden(p4)), 'promise');
```

Comment:
The change to this test case demonstrates the loosening of the `@@toStringTag` tests